### ISSUE TITLE
Still getting duplicated or undesired warnings for missing settings

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#200]
```
*** ATTENTION ***
Your devon-ide is missing the settings at
Please run the following command to complete your IDE setup:
devon ide setup [<settings-url>]
devon-ide environment variables have been set for C:\projects\cool-project
```
This also happens if the settings (here `C:\projects\cool-project\settings`) actually exist.

**Comment by hohwille** [comment]
As you can see here:
```
Your devon-ide is missing the settings at
```
and here:
https://github.com/devonfw/devon-ide/blob/c1eea27ef10e9cc21dc5a870f31d74f6e8efac88/scripts/src/main/resources/scripts/environment-project.bat#L30

The actual bug is that `SETTINGS_PATH` is undefined in this case.

**Comment by hohwille** [comment]
The root cause after debugging seems to be that in this case `DEVON_IDE_HOME` is not set.
As a consequence all `devon.properties` are not found and hence also `SETTINGS_PATH`is not set.

**Comment by hohwille** [comment]
Funny. The problem is that `devon.bat` has changed and once installed it does not get updated and overwritten. Seems we need the same check we introduced for linux/mac also for windows that checks if we have a newer version of `devon` script and then overwrites the older one.
However, in the future `devon` script needs to stay stable and should not change anymore if possible...

**Comment by hohwille** [comment]
See also #177

**Comment by hohwille** [comment]
Fixed with PR #201
